readme.md

```diff
@@ -47,6 +47,12 @@ const findUp = require('find-up');
 
 	console.log(await findUp(['rainbow.png', 'unicorn.png']));
 	//=> '/Users/sindresorhus/unicorn.png'
+
+	console.log(await findUp(dir => Promise.resolve('unicorn.png')));
+	//=> '/Users/sindresorhus/unicorn.png'
+
+	console.log(await findUp(dir => dir));
+	//=> '/Users/sindresorhus'
```

Comment:
Can you add more realistic examples? I think one using `fs.existsSync` and custom logic for deciding whether it should be used or not would be good. To give users an indication about the capabilities.

Reply:
Done. I think I was trying to avoid the noise of importing extra modules in the example. But yeah, that was a silly example.